src/components/UnderConstruction.tsx

```diff
@@ -1,9 +1,42 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Image from "next/image";
 import { FaWhatsapp } from "react-icons/fa";
+import Link from 'next/link';
 import styles from "@/styles/components/UnderConstruction.module.css";
 
 function UnderConstruction() {
+
+  const [days, setDays] = useState(0)
+  const [hours, setHours] = useState(0)
+  const [minutes, setMinutes] = useState(0)
+  const [seconds, setSeconds] = useState(0)
+
+  useEffect(() => {
+    const target = new Date('03/10/2023 23:59:59')
+
+    const interval = setInterval(() => {
+      const now = new Date()
+      const difference = target.getTime() - now.getTime()
```

Comment:
Add a condition to handle the case when the difference is 0 or negative to prevent negative results